atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/pool/CassandraService.java

```diff
@@ -126,7 +126,7 @@ public TokenRangeWritesLogger getTokenRangeWritesLogger() {
             tokenRangeWritesLogger.updateTokenRanges(tokenMap.asMapOfRanges().keySet());
             return servers;
         } catch (Exception e) {
-            log.error("Couldn't grab new token ranges for token aware cassandra mapping!", e);
+            log.warn("Couldn't grab new token ranges for token aware cassandra mapping!", e);
```

Comment:
The main implication of this is that we lose the perf optimisation we have where we talk directly to the correct nodes, right? If that is correct, `warn` is probably fine

Reply:
Is this something that can be/is retried? If so, then the problem is likely to fix itself, possibly without us even realising that it has done so (as I don't think we log when the token ranges were successfully updated, unless it's logged somewhere else).

Again, if `refreshTokenRanges` is only called occasionally, we could consider a separate PR where we periodically retry the mapping update in the event of a failure. This can then be an `info`, as human action will probably not be required to remediate the issue.

Reply:
It's retried every 2 minutes, so I'm bumping it to info and adding that detail to the message.